cdap-docs/admin-manual/source/hadoop-compatibility.rst

```diff
@@ -9,27 +9,31 @@ Hadoop Compatibility
 ====================
 
 Before installing the CDAP components, you must first install (or have access to) a Hadoop
-cluster with *HDFS*, *YARN*, *HBase*, and *ZooKeeper*. In order to use the ad-hoc querying
-capabilities of CDAP (Explore), you will also need *Hive*. All CDAP components can be installed on
-the same boxes as your Hadoop cluster, or on separate boxes that can connect to the Hadoop
-services.
+cluster with *HBase*, *HDFS*, *YARN*, and *ZooKeeper*. In order to use the ad-hoc querying
```

Comment:
Hive and Spark are optional components; hive is required to enable ad-hoc querying capabilities and spark is required if the CDAP application uses spark program.